### PR TITLE
Feature#41 styles singup

### LINKED ISSUE #41: Add styles for teacher sing up
# REQUIREMENTS
Add the styles for the teacher record view.
Reload the form a little on the right and put a brief description of the role on the left side

# Acceptance Criteria
- [ ] styles teacher sing up

### PR DESCRIPTION
**Pull request by RobertoCarMed**
# Description
Styles were added for the teacher record view:
- Two images were added for the desktop layout
- Styles for mobile were added
- Styles for desktop size added
  
  ## Screenshots 
image
image
image

resolve #41

### CODE REVIEW

File: app/assets/stylesheets/teachers/registrations/components/_new_table.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.container{
+
+  .form-teacher{
+    width:65%;
+    
+
```
Comment (xFerchoVJ) [inline app/assets/stylesheets/teachers/registrations/components/_new_table.scss:6]: beware of unnecessary spaces

**Review by xFerchoVJ — DISMISSED** [review]
I saw only one detail, but you can fix it in the next PR🥸
Nice job.

**Review by SampabloC — DISMISSED** [review]

**Review by YilKanda — COMMENTED** [review]
You need to delete the unnecessary spaces that are in your style files.

**Review by xFerchoVJ — APPROVED** [review]
LGTM

**Review by OmarRoBra — APPROVED** [review]
Good Job :D

**Review by SampabloC — APPROVED** [review]

File: app/assets/stylesheets/teachers/registrations/components/_new_table.scss (hunk 1 above)
Comment (RobertoCarMed) [reply app/assets/stylesheets/teachers/registrations/components/_new_table.scss]: thanks you

**Pull request merged**